student.rb

```diff
@@ -3,9 +3,10 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
-    super(id, age, name, parent_permission)
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] To correctly complete the `has-many / belongs-to relationships` between `Clasroom` and `Student` classes, kindly ensure that the `Student` class has a method that serves as the `belongs-to` side (a student belongs to a classroom). It should look to something like this:
```ruby
class Student < Person
  attr_reader :classroom
  .
  .
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```